phoenix-core/src/it/java/org/apache/phoenix/end2end/index/PartialIndexIT.java

```diff
@@ -764,4 +764,47 @@ public void testPartialIndexPreferredOverFullIndex() throws Exception {
             assertFalse(rs.next());
         }
     }
+    
```

Comment:
My recommendation would be to modify some of the existing tests in this class and add this additional check there rather than creating a new test.

Reply:
@tkhurana sure. Let me look into it.